04/app.js

```diff
@@ -37,6 +45,15 @@ function getCoordinates() {
   const longitudeInput = formEl.querySelector(".form__field--lng");
   const latitudeValue = latitudeInput.value;
   const longitudeValue = longitudeInput.value;
+  if (!latitudeValue || !longitudeValue) {
+    console.error(
+      "Wartości szerokości i długości geograficznej musza byc podane!"
+    );
+    alert("Wartości szerokości i długości geograficznej musza byc podane!");
+    return Promise.reject(
+      "Wartości szerokości i długości geograficznej musza byc podane!"
+    );
```

Comment:
Raczej wybrałbym jeden ze sposób :p

PS. Jak ma już zostać kilka to bym zapisał komunikat w zmiennej i ją wykorzystywał w poszczególnych elementach - wtedy poprawa takiego komunikatu to jedno miejsce, a nie wiele ;)